Client/mods/deathmatch/logic/luadefs/CLuaResourceDefs.cpp

```diff
@@ -433,6 +433,14 @@ int CLuaResourceDefs::GetResourceState(lua_State* luaVM)
 
 int CLuaResourceDefs::LoadString(lua_State* luaVM)
 {
+    CLuaMain& luaMain = lua_getownercluamain(luaVM);
+
+    if (!luaMain.IsUnsafe())
```

Comment:
We should throw an error here / print an error log, to indicate why this has failed.